fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -250,7 +253,17 @@ private IndexingByIndex getIndexerByIndex() {
                             .build();
                     return indexingLauncher(indexingFunc, attemptCount, origPolicy);
                 }
+
                 // No other methods (yet). This line should never be reached.
+                if (LOGGER.isErrorEnabled()) {
+                    LOGGER.error(KeyValueLogMessage.build("Invalid previous indexing type stamp",
```

Comment:
This feels like it is should be a warn-level log if no error is thrown

Reply:
Fixed.